tests/tests/swfs/avm2/stage3d_raytrace/test.toml

```diff
@@ -5,12 +5,12 @@
 num_frames = 80
 
 [image_comparison]
-tolerance = 0
+tolerance = 1
 # FIXME - see if we can decrease this. Ideally, it wouldn't be nearly so high
 max_outliers = 5100
 
 [player_options]
 # This test runs very slowly on unoptimized `test` builds,
 # so give it plenty of time.
 max_execution_duration = { secs = 1000, nanos = 0 }
-with_renderer = { optional = false, sample_count = 1 }
+with_renderer = { optional = false, sample_count = 1, exclude_warp = true }
```

Comment:
Can you add a link to https://github.com/gfx-rs/wgpu/issues/3193 ?

Reply:
I'll rebase & add that in a comment once I see the test results :D